src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
+  height: 12px;
+  width: 12px;
+  margin: 2px 6px 2px 2px;
+  background-image: url("/images/star.svg");
+  background-size: cover;
+}
+
+.stars__star:nth-child(n) {
+  background-image: url("/images/star.svg");
+}
+
```

Comment:
let's think, here you set the bg for every `.stars__star`, which you have already done in the selector right above

Reply:
Makes sense.